src/templates/default/cbg.mjs

```diff
@@ -0,0 +1,59 @@
+export const cbgDefault = {
+  cbg: function () {
+    return {
+      resourceType: 'Observation',
+      meta: {
+        profile: [
+          'http://phr.kanta.fi/StructureDefinition/fiphr-bloodglucose-stu3-modified20190128',
+        ],
+      },
+      language: 'en',
+      text: {
+        status: 'extensions',
+        div: `<div xmlns="http://www.w3.org/1999/xhtml">Code: Home glucose sensor reading<br />Time: ${this.time_fhir} <br />Device: ${this.deviceId} (via Sensotrend Connect)<br />Result: ${this.valueMmol} mmol/l</div>`,
```

Comment:
Nämä kai pitäisi oikeasti olla suomeksi Omatietovarantoon vietäessä.

Reply:
Ks. esim. tuolta https://github.com/Sensotrend/sensotrend-converter/pull/30/files#diff-648d6074dd80a8b40f6af5145a0b0a3124e8f7fefcf2fd6c11c4687cdbfb2787R11

Reply:
Mutta juu, nämähän ovat "default"-templateja, eli nimeomaan Omatietovarannolle on sitten omansa. Eli olkoon näin.